Aspose.Slides.WebExtensions/Helpers/SlideHelper.cs

```diff
@@ -4,12 +4,73 @@
 using System.Collections.Generic;
 using System.Drawing;
 using Aspose.Slides.Animation;
+using Aspose.Slides.Export.Web;
 using Aspose.Slides.SlideShow;
 
 namespace Aspose.Slides.WebExtensions.Helpers
 {
     public static class SlideHelper
     {
+        public static string GetNextSlide(TemplateContext<Slide> Model)
+        {
+            IPresentation presentation = Model.Object.Presentation;
+            Slide current = Model.Object as Slide;
+            string result = "";
+            if (presentation != null && presentation != null)

```

Comment:
@polonair ,

Seems like the second check issue.. it might be smth like that:

```suggestion
            if (presentation != null && current != null)
```
?

Reply:
Done